src/ci/docker-compose/cromwell-test/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM openjdk:8
+FROM adoptopenjdk:8-jre-hotspot
```

Comment:
I thought we could "adopt" (I don't know the right term) the images created by appsec and build on top of them

Mildly preferable if we don't have to check in and then maintain `apt` commands to Cromwell